tests/System.Time.Tests/DateTimeOffsetTests.cs

```diff
@@ -7,10 +7,17 @@ namespace System.Time.Tests
 {
     public class DateTimeOffsetTests
     {
+
+#if Windows
+    string TimeZoneId = "Pacific Standard Time";
+#else
+    string TimeZoneId = "America/Los_Angeles";
```

Comment:
These strings should work for Linux and Mac. Still, I will see if I can run the tests on Mac as well.

https://stackoverflow.com/tags/timezone/info

> The IANA/Olson Time Zone Database
Widely implemented on Linux, Mac, Java, PHP, and many other platforms.